vest/src/regular/error.rs

```diff
@@ -0,0 +1,69 @@
+use crate::properties::*;
+use vstd::prelude::*;
+
+verus! {
+
+/// Combinator used for custom error message
+pub struct CustomError(pub String);
+
+impl View for CustomError {
```

Comment:
A bit nitpicking, personally I prefer the name "`Fail`" to indicate that this combinator would always fail. `CustomError`  per se doesn't sound like a combinator. If found useful, in future we might also introduce a "`Succeed`" combinator that always succeeds and produces no values.

Reply:
Yeah Fail is a better name. I was just trying to avoid Error since it overlaps with the error type.